### PR TITLE
#52 - Create-card component

### LINKED ISSUE #52: Create a CreateCard component
Props
- icon?: IconProps, Default: Plus Icon
- text: TextProps

### PR DESCRIPTION
**Pull request by Koushith**
## Description
- [x] Adds Reusable Create Card component.
- [x] Removed Test Data from Home Screen which was pushed accidentally. 

## Screenshot

image

## Platform tested
- [x] Web

### CODE REVIEW

File: src/components/primitive/create-card/create-card.component.styles.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    height: 24.3rem !important;
+    max-width: 33rem !important;
+    margin: 1.5rem 0 1.5rem !important;
+  }
+`;
+
+/**
+ * Styles for text inside the cards.
+ */
+export const SafeText = styled(Text)`
```
Comment (dsouzaedison) [inline src/components/primitive/create-card/create-card.component.styles.ts]: This is a generic card. It cannot have reference to business logic. Rename it to Label or something.

File: src/i18n/locales/en.json [hunk 2]
```diff
@@ -2,8 +2,14 @@
...
-    "Profile": "Profile"
+    "home": "Home",
+    "claims": "Claims",
+    "profile": "Profile",
+    "safes": "Safes",
+    "createSafe": "Create Safe",
+    "genericNotes": "Generic Notes",
+    "safeRole": "Safe Role",
+    "Creator": "Creator",
+    "Safients": "Safients"
```
Comment (dsouzaedison) [inline src/i18n/locales/en.json]: Property names must be in small cases!

File: src/screens/home/home.screen.tsx [hunk 3]
```diff
@@ -1,18 +1,10 @@
...
-  padding: 10px;
-  margin: 10px;
-`;
 export const HomeScreen: React.FC = () => {
   return (
-    <div>
-      <NavBar />
-      {/* <BottomTab /> */}
-    </div>
+    <>
```
Comment (dsouzaedison) [inline src/screens/home/home.screen.tsx:6]: This wrapper is not required

File: src/screens/home/home.screen.tsx [hunk 4]
```diff
@@ -1,18 +1,10 @@
...
+      <Text variant='title'>Dashboard</Text>
```
Comment (dsouzaedison) [inline src/screens/home/home.screen.tsx]: Why is this string hard-coded?

**Review by dsouzaedison — CHANGES_REQUESTED** [review]

File: src/screens/home/home.screen.tsx (hunk 4 above)
Comment (Koushith) [reply src/screens/home/home.screen.tsx]: That was the Test Data that was pushed accidentally. I have cleared that in #20  Home Screen PR.

File: src/screens/home/home.screen.tsx (hunk 3 above)
Comment (Koushith) [reply src/screens/home/home.screen.tsx:6]: Removed in #20 HomeScreen PR.

**Review by dsouzaedison — APPROVED** [review]

**Pull request merged**